meta/src/grammar.pest

```diff
@@ -171,6 +171,6 @@ space       = _{ " " | "\t" }
 /// A top-level comment.
 grammar_doc = ${ "//!" ~ space? ~ inner_doc }
 /// A rule comment.
-line_doc    = ${ "///" ~ space? ~ !"/" ~ inner_doc }
```

Comment:
Before this rule was following the Rust `line_doc` rule, wants to be let the `////` as a normal comment.

But I think, maybe we don't need to do that, just let it as a `line_doc` is also ok. So it will not come to a complex rule.